arranger/kustomization.yml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ingresses.yml
+  - namespace.yml
+  - services.yml
+
+patchesStrategicMerge:
+  - configmaps.yml
+  - deployments.yml
```

Comment:
I think you want to put those two entries under "resources". "patchesStrategicMerge" is used when you want to "add to" and/or "overwrite" entries in a resource defined previously, but here, you are defining those resources for the first time.